app.territoiresentransitions.react/src/ui/shared/form/useAutoSizeTextarea.ts

```diff
@@ -17,7 +17,7 @@ export const useAutoSizeTextarea = (
   // Permet de set la taille du textarea au changement de valeur
   useLayoutEffect(() => {
     if (usedRef && usedRef.current) {
-      usedRef.current.style.height = '0px';
```

Comment:
J'ai vraiment du mal à reproduire le bug, juste 2x en testant pendant 5min.

En fouinant un peu je suis tombé sur ce commentaire qui parle de comportement bizarre avec le scroll sur certains navigateurs.
https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas/#comment-1794830
Il utilise un deuxième élément pour garder une hauteur quand il reset la hauteur, l'équivalent de notre `height: 0`

Je suis allé voir sur MUI comment ils font et eux aussi utilisent un deuxième élément caché
https://github.com/mui/material-ui/blob/master/packages/mui-base/src/TextareaAutosize/TextareaAutosize.tsx

Peut être à voir de ce côté là pour